### PR TITLE
resprites the commander

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
title

Was meant to be apart of another big manufacturer PR, however I do those when i'm taking a break from big projects, so its a bit of a cycle. It appears im in the latter half of the cycle (overmap 5 coming december...), so might as well just PR what i have done

## Why It's Good For The Game

image

## Changelog

:cl:
add: Resprites the commander and Commissioner
/:cl:

### CODE REVIEW

**Review by Imaginos16 — APPROVED** [review]
Neat

**Comment by Aquidu** [comment]
as a diehard commander user i approve this

File: code/modules/projectiles/guns/manufacturer/nanotrasen_sharplite/ballistics.dm [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
```
Comment (thgvr) [inline code/modules/projectiles/guns/manufacturer/nanotrasen_sharplite/ballistics.dm]: how does this even pass linters. no newlines at top of file

Comment (thgvr) [inline code/modules/projectiles/guns/manufacturer/nanotrasen_sharplite/ballistics.dm]:
```suggestion

```

**Pull request merged**